pages/api/roles/index.ts

```diff
@@ -1,5 +1,129 @@
+import { isEmpty } from '@firebase/util';
```

Comment:
Sorry, did you mean this?

```suggestion
import { isEmpty } from '../../../../../utils/strings/validations';
```